templates/takeovers/_fing_webinar.html

```diff
@@ -0,0 +1,57 @@
+<section data-lang="en" lang="en-gb" class="p-strip--dark is-deep p-takeover--fing-webinar js-takeover {% if  default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1 class="p-takeover__title">Future-proofing Fingbox the home network monitoring device</h1>
+      <p class="p-takeover__text">Learn how Fing uses Ubuntu Core and Snaps to speed development, save budget and resources and ensure a future-proofed product.</p>
+      <img class="p-takeover__image-small u-hide--medium u-hide--large" alt="Fing's Fingbox" src="{{ ASSET_SERVER_URL }}3811386c-fingbox_w-shadow.png" />
+      <p class="p-button--neutral"><a href="/engage/future-proof-iot" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'fing webinar takeover', 'eventLabel' : 'Future-proofing Fingbox the IoT home network monitoring device', 'eventValue' : undefined });" style="color: #000;">Register for the webinar</a></p>
```

Comment:
UTMs are missing, should be /engage/future-proof-iot?utm_medium=takeover&utm_campaign=FY19_IOT_WBN_Fing